source/3d-secure-2.html.erb

```diff
@@ -0,0 +1,132 @@
+              <a class="govuk-link" href="/take-payments-by-phone-or-post" itemprop="item"><span itemprop="name">Take payments by phone or post</span></a>
+            </li>
+            <li class="sub-navigation__item" itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem">
+              <a class="govuk-link" href="/payment-service-provider" itemprop="item"><span itemprop="name">GOV.UK Pay’s Payment Service Provider</span></a>
+            </li>
+            <li class="sub-navigation__item" itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem">
+              <a class="govuk-link" href="/direct-debit" itemprop="item"><span itemprop="name">Direct Debit</span></a>
+            </li>
+            <li class="sub-navigation__item" itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem">
+              <a class="govuk-link" href="/apple-pay-and-google-pay" itemprop="item"><span itemprop="name">Apple Pay and Google Pay</span></a>
+            </li>
+            <li class="sub-navigation__item" itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem">
+              <a class="govuk-link" href="/roadmap" itemprop="item"><span itemprop="name">Roadmap</span></a>
+            </li>
+            <li class="sub-navigation__item sub-navigation__item--active" itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem">
+              <a class="govuk-link" href="/security" itemprop="item"><span itemprop="name">Security and compliance</span></a>
+            </li>
+          </ol>
+        </nav>
+      </div>
+
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-l">Strong Customer Authentication: what it is and how it will work</h1>
+      <p class="govuk-body">Strong Customer Authentication (SCA) will make online payments more secure for users and reduce fraud. It is part of the Payment Service Directive (PSD2) as enforced by the Financial Conduct Authority.</p>
+      <p class="govuk-body">Services that take payments from the EU, Norway, Iceland and Liechtenstein (the European Economic Area, EEA), need to implement SCA by December 2020. Services that only take payments from within the UK need to implement SCA by September 2021.</p>
+      <p class="govuk-body">3D Secure 2 (3DS2) adds an extra layer of identity confirmation before the user pays so that transactions meet the SCA rules.</p>
+      <p class="govuk-body">3DS2 replaces 3D Secure (3DS). 3DS transactions offer a similar level of protection for users and are compliant with SCA requirements, but introduce more friction to the users’ journey. 3DS will also be phased out, though the date for this has not been confirmed.</p>
+
+      <h2 class="govuk-heading-m">What 3D Secure 2 means for users</h2>
+      <p class="govuk-body">3DS2 will ask users for 2 out of 3 pieces of information to complete their transaction:</p>
```

Comment:
can we link this information with the 2 factor authentication requirement in line 69. For example, "3DS2 requires 2 factor authentication, and will ask users for 2 out of 3 pieces of information to complete their transaction"

If it isn't as simple as that, no change needed